app/com/lynxanalytics/biggraph/partitioned_parquet/PartitionedParquet.scala

```diff
@@ -80,6 +80,7 @@ class PartitionedParquetScan(
     readPartitionSchema: StructType,
     pushedFilters: Array[Filter],
     options: CaseInsensitiveStringMap,
+    pushedAggregate: Option[org.apache.spark.sql.connector.expressions.aggregate.Aggregation] = None,
```

Comment:
This comes from https://github.com/apache/spark/commit/128168d8c4019a1e10a9f1be734868524f6a09f0 ([SPARK-36645](https://issues.apache.org/jira/browse/SPARK-36645)).